pkg/client/supplier/client.go

```diff
@@ -143,6 +143,10 @@ func (sClient *supplierClient) CreateClaims(
 	return <-errCh
 }
 
+func (sClient *supplierClient) Address() *cosmostypes.AccAddress {
```

Comment:
1. Is this being used anywhere?
2. godoc please

Reply:
1. [Yes](https://github.com/pokt-network/poktroll/blob/63c67681d80079327153b5348bac9ee35569423a/pkg/relayer/cmd/cmd.go). This allows us to use account address in the supplier client map instead of a key name. If we were to use the key name here, we'd need to resolve it to the address on multiple places.
2. On it!